### PR TITLE
dxScheduler. TestCafe. Resize appointments and common code refactoring

### PR DESCRIPTION
**Pull request by ghost**
Minimizing code by adding the same code in a loop of different dxScheduler views

### CODE REVIEW

File: testing/functional/tests/scheduler/resizeAppointments/timeline.ts [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+import { createScheduler } from './init/widget.setup';
+import url from '../../../helpers/getPageUrl';
+import Scheduler from '../../../model/scheduler';
+
+fixture `Resize appointments in the Scheduler basic mode`
+	.page(url(__dirname, '../../container.html'));
+
+['timelineDay', 'timelineWeek', 'timelineWorkWeek'].forEach((mode) => test(`Resize in the "${mode}" mode`, async t => {
+	const scheduler = new Scheduler("#container");
+	const resizableAppointment = await scheduler.getAppointment('Brochure Design Review');
```
Comment (churkin) [inline testing/functional/tests/scheduler/resizeAppointments/timeline.ts]: extra `await`

**Review by churkin — CHANGES_REQUESTED** [review]

File: testing/functional/tests/scheduler/resizeAppointments/init/widget.setup.ts [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+export const startDayHour = 9;
+export const currentDate = new Date(2019, 3, 1);
+
+export const resources = [
+	{ id: "0", color: "#e01e38" },
+	{ id: "1", color: "#f98322" },
+	{ id: "2", color: "#1e65e8" }
+];
+
+export const createScheduler = async (mode, dataSource, firstDayOfWeek = 1) => {
```
Comment (churkin) [inline testing/functional/tests/scheduler/resizeAppointments/init/widget.setup.ts]: remove `async` and `await` please :)

**Review by churkin — DISMISSED** [review]

**Review by churkin — DISMISSED** [review]

File: testing/functional/tests/scheduler/resizeAppointments/timeline.ts (hunk 1 above)
Comment (ghost) [reply testing/functional/tests/scheduler/resizeAppointments/timeline.ts]: Complete.

File: testing/functional/tests/scheduler/resizeAppointments/init/widget.setup.ts (hunk 2 above)
Comment (ghost) [reply testing/functional/tests/scheduler/resizeAppointments/init/widget.setup.ts]: Complete.

**Review by churkin — DISMISSED** [review]

File: testing/functional/tests/scheduler/dragAndDrop/basic.ts [hunk 3]
```diff
@@ -4,52 +4,28 @@ import url from '../../../helpers/getPageUrl';
 import Scheduler from '../../../model/scheduler';
 
 fixture `Drag-and-drop appointments in the Scheduler basic mode`
-    .page(url(__dirname, '../../container.html'));
+	.page(url(__dirname, '../../container.html'));
```
Comment (Olexi) [inline testing/functional/tests/scheduler/dragAndDrop/basic.ts]: First 'tabulation' symbol must be 4 spaces, not 3 (as here).

File: testing/functional/tests/scheduler/resizeAppointments/init/widget.data.ts [hunk 4]
```diff
@@ -0,0 +1,23 @@
+export const dataSource = [
+    {
+        Id: 1,
+        text: 'Brochure Design Review',
+        startDate: new Date(2019, 3, 1, 10, 0),
+        endDate: new Date(2019, 3, 1, 11, 0),
+        resourceId: '0'
```
Comment (Olexi) [inline testing/functional/tests/scheduler/resizeAppointments/init/widget.data.ts]: resourceId must be `number`, not `string`

File: testing/functional/tests/scheduler/resizeAppointments/init/widget.setup.ts [hunk 5]
```diff
@@ -0,0 +1,33 @@
+import { createWidget } from "../../../../helpers/testHelper";
+
+export const widgetHeight = 833;
+export const widgetWidth = 1402;
+export const maxAppointmentsPerCell = 5;
+export const startDayHour = 9;
+export const currentDate = new Date(2019, 3, 1);
+
+export const resources = [
+	{ id: "0", color: "#e01e38" },
```
Comment (Olexi) [inline testing/functional/tests/scheduler/resizeAppointments/init/widget.setup.ts]: id must be `number`

File: testing/functional/tests/scheduler/resizeAppointments/init/widget.setup.ts [hunk 6]
```diff
@@ -0,0 +1,33 @@
+import { createWidget } from "../../../../helpers/testHelper";
+
+export const widgetHeight = 833;
```
Comment (Olexi) [inline testing/functional/tests/scheduler/resizeAppointments/init/widget.setup.ts]: `const` should not be separated from the initialization of Scheduler, 'cause of single-use of it

**Review by Olexi — CHANGES_REQUESTED** [review]

**Review by churkin — DISMISSED** [review]

**Review by Olexi — APPROVED** [review]

**Review by andrewmakarov — APPROVED** [review]

**Review by churkin — APPROVED** [review]

**Review by jsdmitry — APPROVED** [review]

**Review by IgnatovDan — APPROVED** [review]

**Pull request merged**